pkg/cloudprovider/providers/gce/gce_disks_test.go

```diff
@@ -30,10 +32,13 @@ func TestCreateDisk_Basic(t *testing.T) {
 	/* Arrange */
 	fakeManager := newFakeManager()
 	projectId := "test-project"
+	zonesWithNodes := []string{"zone1"}
```

Comment:
this whole file was mangled in rebase so redone from scratch. please look at more carefully.